spec/factories/intakes.rb

```diff
@@ -321,7 +321,7 @@
 
   trait :with_faker_contact_info do
     primary_first_name { Faker::Name.first_name }
-    primary_last_name { Faker::Name.last_name }
```

Comment:
I figured if we want to be sure we handle apostrophes reliably, we can choose to always add an apostrophe in there.

Reply:
I find Faker really irritating because in feature-land someone will write a test that says `expect(page).to have_content(last_name)` and it will fail one out fifty times because what's on the page is `O&apos;Connor` or whatever

I don't see a point to making our test data this random, and I don't like having to care about apostrophes when making those kind of assertions, so if we're going to change `primary_last_name` to anything I think it should be a nice static string like "Smith"